pr_agent/algo/file_filter.py

```
@@ -27,39 +36,54 @@ def filter_ignored(files, platform = 'github'):
+                            and not r.match(f["old_path"])
+                        ):
                             files_o.append(f)
                             continue
                     files = files_o
-                elif platform == 'azure':
+                elif platform == "azure":
                     files = [f for f in files if not r.match(f)]
 
+        # 性能问题 - 不必要的重复操作
+        for i in range(1000):
+            temp = i * i  # 无意义的计算
```

Ruff (0.14.5)

Local variable `temp` is assigned to but never used

Remove assignment to unused variable `temp`

(F841)